lib/read_clover.h

```diff
@@ -187,40 +187,40 @@
   C1.y += param.rho; C2.x += param.rho; C2.y += param.rho;
 
 #define READ_CLOVER_SINGLE_TEX(clover, chi)			\
```

Comment:
Broad file-wide comment---this is missing "QUARTER" versions of these routines, and presumably wherever these routines get called are also missing quarter versions.

Reply:
This is a legacy macro file that will be deleted.

Reply:
:+1: